src/main/java/Task.java

```diff
@@ -1,2 +1,26 @@
-package PACKAGE_NAME;public class Task {
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description)
+    {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public Task()
+    {
+        this.description = "empty";
+        this.isDone = false;
+    }
+
+    public void markAsDone()
+    {
+        this.isDone = true;
+    }
+
+    public String getStatusIcon()
+    {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
```

Comment:
The first "{" of each method should be next to the name of method.
eg. public String getStatusIcon() {
       ......
}